base_multi_image/views/owner_templates.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<!-- Call this template in your owner's website page to get a carousel.
+     It needs a `multi_image_owner` variable set. -->
```

Comment:
See usage example in https://github.com/OCA/event/pull/33/files#diff-068a0c4a89bc9582a1c4cf77d01be469R32